src/pyedb/siwave.py

```diff
@@ -509,16 +518,17 @@ def load_configuration(self, file_path: str):
         # temp_folder = tempfile.TemporaryDirectory(suffix=".ansys")
         # temp_edb = os.path.join(temp_folder.name, "temp.aedb")
 
-        temp_edb = os.path.join(self.file_dir, "temp.aedb")
+        temp_edb = os.path.join(tempfile.gettempdir(), generate_unique_name("temp") + ".aedb")
 
         self.export_edb(temp_edb)
         self.oproject.ScrCloseProject()
-        edbapp = Edb(temp_edb, edbversion=self.current_version)
+        edbapp = Edb(temp_edb, edbversion=self.version)
         edbapp.configuration.load(file_path)
         edbapp.configuration.run()
         edbapp.save()
         edbapp.close()
         self.import_edb(temp_edb)
+        shutil.rmtree(Path(temp_edb))
```

Comment:
```suggestion
        shutil.rmtree(Path(temp_edb), ignore_errors=True)
```
This will avoid test failure ok cleanup (if that ever happens).